app/src/App.jsx

```diff
@@ -2,11 +2,14 @@ import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import Homepage from './components/HomePage';
 // import ProfilePage from './components/profilePage';
 import ProfilePage from './components/ProfilePage';
+import Nav from './components/Nav'
 
 function App() {
   return (
     <>
+    <Nav/>
```

Comment:
two browser routers next to each other, why not place nav inside browser router and return a routes element